### ISSUE TITLE
Player slider should be a focusable control

### ISSUE THREAD

**Issue by chrisauclair** [cs50/video50#88]
Keyboard users should be able to select the slider and navigate to set points using the left/down and right/up arrows.

**Comment by coltonoscopy** [comment]
@chrisauclair There doesn't seem to be a slider anymore (maybe there was when this was first proposed in November?). There's now a red progress bar of sorts that takes up the entire timeline based on how far the viewer is in the video. This bar is focusable, but using the keyboard atm just progresses the video forward a few seconds per YouTube's API. Does not using the keyboard to navigate through the chapters in the sidebar and then select those to progress on the timeline in the same way accomplish this goal? I argue this way is even better because the viewer can see specifically which chapter they're moving to vs. looking at blips on the timeline!

**Comment by chrisauclair** [comment]
@coltonoscopy, the player has changed a lot since I last looked at it. In most video player implementations, the seek slider is a selectable element, and users can press the right or left arrow keys to seek `x` number of seconds determined by both the length of the video and the width of the slider. I recommend that you try the YouTube player using keyboard controls for a great example.

Unfortunately I don't have an answer on how to mix that type of seek slider with chapters. You could mix the chapters and the more typical time-based seek points in the same seek bar, but that could make it worse. My gut instinct is to avoid these types of multimodal controls from a usability standpoint.

**Comment by coltonoscopy** [comment]
@chrisauclair, the player currently just inherits YouTube's default skip behavior (5 seconds ahead and back on arrow key press). Is this what you mean?

@dmalan, would you like to weigh in?

**Comment by coltonoscopy** [comment]
Closing per @lukejacksonn's latest changes, which allow the chapters on the timeline to be tabbable; users can now skip ahead and back in 5-second chunks via YouTube's API as before and also per these new larger chunks.
